### PR TITLE
Removed jaggery.js and replaced with jsp for services. Migrated all apim-apps as webapps.

### PR DESCRIPTION
**Pull request by DinithHerath**

### CODE REVIEW

**Comment by DinithHerath** [comment]
Merging can not be done untill the PRs for all portals get merged in the apim-apps side.

**Comment by DinithHerath** [comment]
wso2/apim-apps#309

File: modules/distribution/product/src/main/assembly/bin.xml [hunk 1]
```diff
@@ -1584,16 +1568,16 @@
...
-            <source>src/main/conf/lerna/lerna.json</source>
-            <outputDirectory>wso2am-${pom.version}/repository/deployment/server/jaggeryapps</outputDirectory>
-            <destName>lerna.json</destName>
-        </file>
-        <file>
-            <source>src/main/conf/lerna/package.json</source>
-            <outputDirectory>wso2am-${pom.version}/repository/deployment/server/jaggeryapps</outputDirectory>
-            <destName>package.json</destName>
-        </file>
+<!--        <file>-->
```
Comment (tharikaGitHub) [inline modules/distribution/product/src/main/assembly/bin.xml:1571]: Should we remove this completely instead of commenting out?

Comment (DinithHerath) [reply modules/distribution/product/src/main/assembly/bin.xml:1571]: +1

**Review by tharikaGitHub — APPROVED** [review]

**Review by chanaka3d — APPROVED** [review]

**Review by chanaka3d — APPROVED** [review]

**Comment by sajinieKavindya** [comment]
Shall we add a proper description to the PR?

**Pull request merged**